BUILD

```diff
@@ -9,6 +9,7 @@ python_requirements(
     "python3-saml": ["onelogin"],
     "social-auth-app-django": ["social_django"],
     "setuptools": ["pkg_resources"],
+    "edx-opaque-keys": ["opaque_keys"],
```

Comment:
Is there any special reason my this is included  in the projects build file rather then the apps build file

Reply:
Since all the third parties being used are in requirements.txt file, which is placed at project level, not at app level, so module_mapping in python_requirements has also to be at project level hence it is in projects build file instead of apps build file.
More details can be seen here: https://www.pantsbuild.org/docs/python-third-party-dependencies